style.css

```diff
@@ -0,0 +1,1067 @@
+  filter: grayscale(1);
+  transform: rotate(-13deg);
+}
+
+.grid-framework {
+  display: grid;
+  grid-template-columns: 50% 50%;
+  grid-column-gap: 20px;
+}
+
+.java-framwork,
+.css-framework,
+.web-framework,
+.newsletter-framework {
+  color: white;
+  background-color: grey;
+  width: 550px;
+  padding-top: 20px;
+  padding-left: 40px;
+  padding-right: -20px;
+  padding-bottom: 60px;
+  margin-left: 40px;
+  margin-top: 40px;
+  border-radius: 12px;
+}
+
+.css-framework {
+  padding-bottom: 105px;
+  margin-left: -10px;
+}
+
+.web-framework {
+  padding-bottom: 475px;
+}
+
+.newsletter-framework {
+  margin-left: -10px;
+  padding-bottom: 62px;
+}
+
+.java-framwork .head,
+.css-framework .head,
+.web-framework .head,
+.newsletter-framework .head {
+  font-size: 38px;
+  font-weight: 600;
+}
+
+.java-framwork .data,
+.css-framework .data,
+.web-framework .data {
+  line-height: 28px;
+  margin-top: 15px;
+}
+
+.java-btn {
+  height: 55px;
+  width: 410px;
+  border-radius: 8px;
+  font-size: 20px;
+  font-weight: 600;
+  border: none;
+  color: gray;
+  cursor: pointer;
+}
+
+.css-btn {
+  height: 55px;
+  width: 300px;
+  border-radius: 8px;
+  font-size: 20px;
+  font-weight: 600;
+  border: none;
+  color: gray;
+  cursor: pointer;
+}
+
+.explore-btn {
+  height: 55px;
+  width: 280px;
+  border-radius: 8px;
+  font-size: 20px;
+  font-weight: 600;
+  border: none;
+  color: gray;
+  cursor: pointer;
+}
+
+.data span {
+  filter: grayscale(1);
+}
+
+form.newsletter {
+  padding-bottom: 105px;
+}
+
+form.newsletter input[type=text] {
+  padding: 10px;
+  font-size: 17px;
+  border: 1px solid grey;
+  float: left;
+  width: 60%;
+  background: #f1f1f1;
+  border-top-left-radius: 5px;
+  border-bottom-left-radius: 5px;
+  height: 35px;
+}
+
+.newsletter-bottom-para {
+  display: block;
+  background-color: white;
+  color: gray;
+  text-align: center;
+  padding-top: 14px;
+  width: 350px;
+  height: 60px;
+  font-size: 15px;
+  font-weight: 500;
+  margin-left: auto;
+  margin-right: auto;
+  border-radius: 8px;
+}
+
+.all-guides-2 {
+  color: gray;
+  text-align: center;
+  margin-top: 80px;
+  margin-bottom: 100px;
+  border-bottom: 4px solid gray;
+  width: 220px;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.footer-main {
+  background: #696969;
+  margin-top: 0;
+  margin-bottom: 0;
+  padding-top: 50px;
+}
+
+.footer-cont {
+  width: 90%;
+  margin: 0 10px;
+  text-align: center;
+  color: #fff;
+  font-family: Elena, serif;
+}
+
+.footer-cont h4 {
+  margin-bottom: 40px;
+  font-size: 15px;
+  letter-spacing: 1px;
+  word-spacing: 1px;
+}
+
+.lots-ofLinks {
+  line-height: 40px;
+}
+
+.lots-ofLinks li {
+  display: inline-block;
+  list-style: none;
+  font-size: 21px;
+}
+
+.lots-ofLinks a:hover {
+  color: #3d3d3d;
+}
+
+.footer-line1-links {
+  width: 1000px;
+  margin-left: 50px;
+}
+
+.foot-para {
+  width: 450px;
+  margin-top: 60px;
+  margin-bottom: 30px;
+  margin-left: auto;
+  margin-right: auto;
```

Comment:
The properties margin-top, margin-bottom, margin-left, margin-right can be replaced by margin.